GOVERNANCE.md

```diff
@@ -0,0 +1,154 @@
+# Prometheus Governance v1.0
+
+This document describes the rules and governance of the project. It is meant to be followed by all the developers of the project and the Prometheus community. Common terminology used in this governance document are listed below:
+
+* **Team members**: Any members of the private [prometheus-team][team] Google group.
+
+* **Maintainers**: Team members who are leads on an individual project ([MAINTAINERS.md][maintainers.md]).
+
+* **Projects**: Any repository in the Prometheus [GitHub organization][gh].
+
+* **The Prometheus Project**: The sum of all activities performed under this governance, concerning one or more Repositories or the community.
+
+## Values
+
+The Prometheus developers and community are expected to follow the values defined in the [CNCF charter][charter], including the [CNCF Code of Conduct][coc]. Furthermore, the Prometheus community strives for kindness, giving feedback effectively and building a welcoming environment. The Prometheus developer community strives to operate by rough consensus for the majority of decisions and only resort to conflict resolution through a motion if consensus cannot be reached.
+
+## Projects
+
+Each project must have a MAINTAINERS.md file with at least one maintainer. Where a project has a release process, access and documentation should be such that more than one person can perform a release. Releases should be announced on the prometheus-users mailing list. Any new projects should be first proposed on the [developers mailing list][devs] following the voting procedures listed below. When a project is no longer relevant, it should be moved to the prometheus-junkyard Github organization.
+
+## Decision making
+
+### Team members 
+
+Team member status may be given to those who have made ongoing contributions for at least 3 months to the Prometheus Project. This is usually in the form of code improvements and/or notable work on documentation, but organizing events or user support could also be taken into account. 
```

Comment:
I'd consider code as the primary, docs/support/events are the secondary. We've yet to have anyone added just for docs.

Reply:
I value documentation and community work as highly as "code", but I don't want to open another huge discussion about this. I'll move documentation into the "also taken into account" listing.

Reply:
I don't want to spawn an argument either but IMHO second-classing documentation is a mistake. I was a core team member of two projects where my contribution was 80% docs. The resulting work was hundreds of commits and thousands of lines of documentation. The workload of writing docs, reviewing PRs, editing, etc was as much as any other maintainer. Others mileage may vary. I don't feel as strongly about community but definitely think a few folks will tell you the same there. Just my ten cents worth.

Reply:
I feel the same. so … what now?

Reply:
It looks like more people want to leave it as-is than change it.

Reply:
I reverted it.